tests/kafkatest/services/streams.py

```diff
@@ -562,6 +568,8 @@ def prop_file(self):
                       consumer_property.SESSION_TIMEOUT_MS: 60000}
 
         properties['input.topic'] = self.INPUT_TOPIC
+        # TODO KIP-441: consider rewriting the test for HighAvailabilityTaskAssignor
+        properties['internal.task.assignor.class'] = "org.apache.kafka.streams.processor.internals.assignment.StickyTaskAssignor"
```

Comment:
These will become follow-on tasks to fix each test. Thankfully, there aren't many.